### ISSUE TITLE
Join forces?

### ISSUE THREAD

**Issue by sburns** [jacobian/slackline#1]
I published [djangobot](https://github.com/djangobot/djangobot) earlier this week. It and slackline do a lot of things similarly (though I wrote my own mediocre slack API wrapper). The biggest thing that djangobot does above slackline is retrieving messages on the `slack.send` channel and relaying them back over the websocket to slack. I'm not a fan that it's currently a hard-coded `slack.send` channel. I think ideally a reply channel should be setup and then on the django side consuming the `slack.hello` (or `slack.rtm.hello` in your case) event should add the reply channel to a `Group('slack')`.

Regardless, glad to see someone else thinking along these lines, there are some really powerful things that could be done w/ an adaptor like this.

**Comment by jacobian** [comment]
Yes, let's merge things! I'll send you some pull requests that represent the delta between djangobot and slackline, and we can discuss whether to integrate them over there. And I'll close down this library, thanks!
